src/styles/style__header.scss

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: block;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  line-height: 14px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
```

Comment:
You should have separate styles file per block.

Reply:
ready